README.rst

```diff
@@ -65,8 +65,8 @@ gcm
   to contain the 'Server key', which can be acquired from Firebase Console at:
   ``https://console.firebase.google.com/project/<PROJECT NAME>/settings/cloudmessaging/``
 
-Pusher `data` configuration
-=======
+Pusher ``data`` configuration
+-----------------------------
```

Comment:
unlike your new section, this section doesn't belong under `Setup`. You need to undo this and move the new section up.

Reply:
Suppose I should have checked / left as-is but felt very 'Setup'-y to me.